src/js/filter/FilterContainer.js

```diff
@@ -13,8 +13,14 @@ export default React.createClass({
     componentDidMount: function() {
         // list available filters (TODO: only for current data? e.g. plate)
         let url = window.PARADE_FILTERS_URL;
-        if (this.props.plateId) url += '?plate=' + this.props.plateId;
-        if (this.props.datasetId) url += '?dataset=' + this.props.plateId;
+        if (this.props.plateId) {
+            url += '?plate=' + this.props.plateId;
+        }
+        else if (this.props.datasetId) {
+            url += '?dataset=' + this.props.plateId;
```

Comment:
datasetId

Reply:
Why datasetId? We don't tend to use *Id in query strings elsewhere.

Reply:
Ooops - sorry. I see it.